localstack/aws/api/core.py

```diff
@@ -35,13 +35,15 @@ class ServiceException(Exception):
     sender_fault: bool
     message: str
 
-    def __init__(self, *args):
+    def __init__(self, *args, **kwargs):
         super(ServiceException, self).__init__(*args)
 
         if len(args) >= 1:
             self.message = args[0]
         else:
             self.message = ""
+        for key, value in kwargs.items():
+            setattr(self, key, value)
```

Comment:
i approve this change :-) coincidentally had this thought here: https://github.com/localstack/localstack/pull/6875#discussion_r979286936

@bentsku will also make your life easier